esp/templates/program/modules/programprintables/options.html

```diff
@@ -51,13 +51,47 @@ <h3>Receipts</h3>
 
 <h3>Admin Binder Stuff</h3>
 <ul>
-<li><a href="./classesbytime" title="Classes by Time">Class Sections sorted by time</a>
+<li><a href="./classesbytime" title="Classes by Time">Class Sections Sorted by Time</a>
     <ul>
     <li><a href="./classesbytime?grade_min=7&grade_max=8" title="Classes by Time">Limit to middle school (grades 7-8)</a></li>
     <li><a href="./classesbytime?grade_min=9&grade_max=12" title="Classes by Time">Limit to high school (grades 9-12)</a></li>
+    <li><a href="./classesbytime?accepted" title="Classes by Time">Accepted Only</a></li>
+    <li><a href="./classesbytime?scheduled" title="Classes by Time">Scheduled Only</a></li>
+    <li><a href="./classesbytime?accepted&scheduled" title="Classes by Time">Accepted and Scheduled Only</a></li>
+    <li><a href="./classesbytime?cancelled" title="Classes by Time">Cancelled Only</a></li>
     </ul>
 </li>
-<li><a href="./classesbytime?cancelled" title="Classes by Time">Cancelled Class Sections</a></li>
+<li><a href="./classesbyid" title="Classes by ID">Class Subjects Sorted by ID</a></li>
+    <ul>
+    <li><a href="./classesbyid?grade_min=7&grade_max=8" title="Classes by ID">Limit to middle school (grades 7-8)</a></li>
```

Comment:
Can you make all the grade limited ones follow the same capitalization pattern as the rest?